### DISCUSSION TITLE
Feature: "not-in" mode for filter function accessed by component reference [DataTable]

### DISCUSSION THREAD

**Discussion by dnyaneshwar2023** [primefaces/community#571] [PrimeReact] [answered]
Hello folks 👋 ,

Do you guys think we should also have "not-in"/"notIn" mode for the filter function that is used from the component reference for the DataTable?

https://primereact.org/datatable/#api.DataTable.methods
<img width="858" alt="image" src="https://github.com/primefaces/community/assets/122769564/5043b109-67fd-4e7b-8260-2d7d4b6246c9">

I was using it in one of my projects and I was hoping this mode to have since there can be use case where I have use case to avoid some items. (That could be achieved through providing all items except that we want to avoid in "in" mode but that seems hacky when I don't have the whole list of data)

I am happy to contribute on it if it's needed.

**Comment by melloware** [comment, marked as answer]
I would open a GitHub issue and Submit a PR if you are so inclined.

**Reply by dnyaneshwar2023** [reply]
Got it.
Can I also raise a PR?

**Reply by melloware** [reply]
Yep! Please do.

**Reply by dnyaneshwar2023** [reply]
Sure, Thanks

**Reply by dnyaneshwar2023** [reply]
I am not create issue on primefaces community but not in primereact repository.
Is that ok to create it in community issues and add link of it in the PR?

**Reply by melloware** [reply]
Create it here: https://github.com/primefaces/primereact/issues

**Reply by dnyaneshwar2023** [reply]
@melloware 
Raised a PR last week. 
Hoping it to be reviewed soon.

Thanks for your help.

**Reply by melloware** [reply]
Its assigned to PrimeTek.  They will eventually review it.

**Comment by dnyaneshwar2023** [comment]
I think it's closed by mistake.  @melloware Can you recheck?
Thanks.

**Reply by melloware** [reply]
This discussion is closed because the GitHub ticket is open.
